### PR TITLE
Responses rework

### PR DESCRIPTION
**Pull request by Levi-Lesches**
A rework of #11 to have `MessageTemplate` be a field in `NetworkerReplyTrigger` instead of inheriting from it. 

Changes: 
- Rename `NetworkerMessageTrigger` --> `NetworkerMessageTriggerLegacy`
- Rename `NetworkerMessageAttachment` --> `NetworkerMessageAttachmentLegacy`
- Create `MessageTemplate` with a `body` field
- Create `MessageTemplateAttachment` to replace `NetworkerMessageAttachment`
- Create `NetworkerReply` to replace `NetworkerMessageTrigger`
- Temporarily adds a link between `NetworkerMessageTriggerLegacy` and `NetworkerReply`

When migrating, each `NetworkerMessageTrigger` is split up into three models: 
- `body` is used to create a `MessageTemplate`
- `attachments` is used to create `MessageTemplateAttachment`s
- an empty `NetworkerReply` is created with a pointer to the legacy `NetworkerMessageTrigger` 

This way all data is preserved, and the rest of the fields can be easily migrated since `NetworkerMessageTriggerLegacy` is inlined into `NetworkerReply`.

### CODE REVIEW

File: mln/models/static.py [hunk 1]
```diff
@@ -324,3 +306,47 @@ def __str__(self):
 class StartingStack(Stack):
 	"""A stack that users start off with in their inventory when they create an account."""
 	item = models.OneToOneField(ItemInfo, related_name="+", on_delete=models.CASCADE)
+
+class MessageTemplate(models.Model): 
+	networker = models.ForeignKey(User, related_name="+", on_delete=models.CASCADE, blank=True, null=True, limit_choices_to={"profile__is_networker": True})
```
Comment (lcdr) [inline mln/models/static.py:311]: As far as I understood it, `MessageTemplate` is also going to be used for module-sent messages that don't involve a networker, so I'd move this to `NetworkerReply` (this also allows us to make it non-null). I'm sorry that I didn't make this clearer on Discord, I should have mentioned that when we were discussing `networker_name`.

File: mln/models/static.py [hunk 2]
```diff
@@ -324,3 +306,47 @@ def __str__(self):
...
+class MessageTemplateAttachment(Stack): 
+	template = models.ForeignKey(MessageTemplate, related_name="attachments", on_delete=models.CASCADE)
+
+	class Meta: 
+		constraints = (models.UniqueConstraint(fields=("template", "item"), name="messsage_template_attachment_unique_template_item"),)
+
+class NetworkerReply(models.Model): 
+	template = models.ForeignKey(MessageTemplate, related_name="+", on_delete=models.CASCADE)
+	message_body = models.ForeignKey(MessageBody, related_name="+", on_delete=models.CASCADE, null=True, blank=True)
+	message_attachment = models.ForeignKey(ItemInfo, related_name="+", on_delete=models.CASCADE, null=True, blank=True)
```
Comment (lcdr) [inline mln/models/static.py:326]: I think we should make it clearer what's the reply here and what's being replied to. `template` -> `reply` seems straightforward, I'm not so sure about `message_body` and `message_attachment`. Google suggests `prompt`, maybe you also have an idea yourself.

File: mln/models/static.py (hunk 1 above)
Comment (Levi-Lesches) [reply mln/models/static.py:311]: Seems reasonable, I was going back and forth on that myself. This does mean that `MessageTemplate` will just have a `body` property, but on the other hand it serves as a common base class for `MessageTemplateAttachment`.

**Comment by Levi-Lesches** [comment]
Added both of your changes: 
- Moved `MessageTemplate.networker` -> `NetworkerReply.networker`
- Renamed `NetworkerReply.message_body` -> `NetworkerReply.trigger_body`
- Renamed `NetworkerReply.message_attachment` -> `NetworkerReply.trigger_attachment`

**Pull request merged**